docs/snippets/all/archetypes/image_send_columns.cpp

```diff
@@ -0,0 +1,50 @@
+// Send multiple images at once using `send_columns`.
+
+#include <numeric>
+#include <rerun.hpp>
+
+int main() {
+    auto rec = rerun::RecordingStream("rerun_example_image_send_columns");
+    rec.spawn().exit_on_failure();
+
+    // Timeline on which the images are distributed.
+    std::vector<int64_t> times(20);
+    std::iota(times.begin(), times.end(), 0);
+
+    // Create a batch of images with a moving rectangle.
+    const size_t width = 300, height = 200;
+    std::vector<uint8_t> images(times.size() * height * width * 3, 0);
+    for (size_t t = 0; t < times.size(); ++t) {
+        for (size_t y = 0; y < height; ++y) {
+            for (size_t x = 0; x < width; ++x) {
+                size_t idx = (t * height * width + y * width + x) * 3;
+                images[idx + 2] = 255; // Blue background
+                if (y >= 50 && y < 150 && x >= t * 10 && x < t * 10 + 100) {
+                    images[idx + 1] = 255; // Turquoise rectangle
+                }
+            }
+        }
+    }
+
+    // Log the ImageFormat and indicator once, as static.
+    auto format = rerun::components::ImageFormat(
+        {width, height},
+        rerun::ColorModel::RGB,
+        rerun::ChannelDatatype::U8
+    );
+    rec.log_static("images", rerun::borrow(&format, 1), rerun::Image::IndicatorComponent());
```

Comment:
from reading later code, it looks like the default is already to borrow one thing?

```suggestion
    rec.log_static("images", rerun::borrow(&format), rerun::Image::IndicatorComponent());
```

Reply:
oh. I should know that, I introduced that just recently. If this doesn't work then I'll make it!

Reply:
the later code borrows several things, easy to miss
But making this work now :)